tests/integration/test_small_deployment_upgrades.py

```diff
@@ -19,10 +19,18 @@
 
 
 OPENSEARCH_ORIGINAL_CHARM_NAME = "opensearch"
-OPENSEARCH_INITIAL_CHANNEL = "2/edge"
+OPENSEARCH_CHANNEL = "2/edge"
+
 MACHINE_ID = 0
 
 
+VERSION_TO_REVISION = {
+    "2.13.0": 91,
+}
+FIRST_VERSION = "2.12.0"
+FIRST_REVISION = 90
```

Comment:
nit: we could perhaps add these in the above dict, and only keep the `FIRST_VERSION` variable

Reply:
I will do an following PR to add "2.14.0". I will merge this one thou

Reply:
Filed ticket: https://warthogs.atlassian.net/browse/DPE-4560 to track that